package.json

```diff
@@ -2,7 +2,7 @@
   "name": "blockstack",
   "version": "0.3.10",
   "description": "The Blockstack Javascript library for identity and authentication.",
-  "main": "lib/index",
+  "main": "dist/blockstack.js",
```

Comment:
I needed to be able to specify my own fork of the library in my package.json. The only way I could get that to work was by changing this field. I was using a webpack bundler myself, and it didn't complain about giving it an already packaged file. I'd have to look deeper to know for sure. Probably fine to just leave this off.